CHANGELOG.md

```diff
@@ -1,11 +1,28 @@
 # CHANGELOG
 
 -----
+## `v4.1.0-beta`
+
+| api                           | version             | note     |
+|:------------------------------|:--------------------|:---------|
+| arm/network                   | 2016-09-01          | updated  |
```

Comment:
I think this is because we don't release individual API packages instead sdk as a whole.